server/graphql/programs/src/queries/document.rs

```diff
@@ -138,20 +81,3 @@ pub fn documents(
             .collect(),
     }))
 }
-
-impl DocumentSortInput {
-    pub fn to_domain(self) -> DocumentSort {
-        let key = match self.key {
-            DocumentSortFieldInput::Name => DocumentSortField::Name,
-            DocumentSortFieldInput::Type => DocumentSortField::Type,
-            DocumentSortFieldInput::Owner => DocumentSortField::Owner,
-            DocumentSortFieldInput::Context => DocumentSortField::Context,
-            DocumentSortFieldInput::Datetime => DocumentSortField::Datetime,
-        };
-
-        DocumentSort {
-            key,
-            desc: self.desc,
-        }
-    }
```

Comment:
Think it can be moved as well but was it actually needed?

Reply:
No, didn't need to be moved since it's only being used in document query, should I move it back?

Reply:
if you don't mind, James noticed that the type crate blocks other crates from building and thus increases build time...

https://github.com/openmsupply/health-supply-hub/pull/214